web/ui/react-app/src/App.css

```diff
@@ -1,185 +1,191 @@
 body {
-    padding-top: 10px; /* TODO remove */
+  padding-top: 10px; /* TODO remove */
 }
 
 button.classic-ui-btn {
-    padding: 10px 0;
+  padding: 10px 0;
 }
 
 .panel {
-    margin-bottom: 20px;
+  margin-bottom: 20px;
 }
 
-.expression-input {
-    margin-bottom: 10px;
```

Comment:
Ah, this was what I was looking for - removing this makes the spacing between the expression input and the tab pane disappear.

Reply:
Yeah, will revert the file and I'll apply only checkbox CSS

Yesterday I spend some time experimenting with eslint end prettier and probably "autofix on save" option makes all those changes

Reply:
Yep, that's what I guessed :)